### DISCUSSION TITLE
(gitextensions) Re-Add support for streams

### DISCUSSION THREAD

**Discussion by BlythMeister** [chocolatey-community/chocolatey-packages#2082] [Ideas]
As part of https://github.com/chocolatey-community/chocolatey-packages/pull/2063 the streams functionality was lost.

This should be re-added

**Comment by AdmiringWorm** [comment]
TBH, I wonder if we ever needed the stream support.

Until there is a need for stream support, I'll say that we leave it for now.


I am interested in hearing other people's opinions on whether streams should be enabled or not though.

**Reply by AdmiringWorm** [reply]
I have created a poll discussion on this (not everyone is probably interested in leaving a comment).

As such I'll close this one in favor of the poll: https://github.com/chocolatey-community/chocolatey-packages/discussions/2184
